app/i18n/messages/en.json

```diff
@@ -251,12 +251,12 @@
   "ReservationCancelModal.cancelNotAllowedInfoOnlinePayment.2": "Cancellation fee: The booking must be cancelled 14 days before the start of the booking at the latest. If the booking is cancelled less than 14 before the start of the booking, the payment is not refunded. Uncancelled bookings will be invoiced.",
   "ReservationConfirmation.feedbackText": "Thank you for using Varaamo! <a href='{href}'>Send feedback</a>.",
   "ReservationConfirmation.reservationCreatedTitle": "Reservation successful",
-  "ReservationConfirmation.reservationManualConfirmationCreatedTitle": "Manual reservation created title --english--",
+  "ReservationConfirmation.reservationManualConfirmationCreatedTitle": "Reservation request has been sent to the resource administrator for approval",
   "ReservationConfirmation.reservationEditedTitle": "Reservation updated",
   "ReservationConfirmation.resourceButton": "Return to premise",
   "ReservationConfirmation.ownReservationButton": "Return to your reservations",
   "ReservationConfirmation.confirmationText": "You will receive email (<a href='mailto:{email}'>{email}</a>) notification for your reservation.",
-  "ReservationConfirmation.confirmationForManualReservationText": "You will receive email (<a href='mailto:{email}'>{email}</a>) notification for your reservation.--english--",
+  "ReservationConfirmation.confirmationForManualReservationText": "You will receive email (<a href='mailto:{email}'>{email}</a>) about additional information regarding your preliminary reservation.",
```

Comment:
Small point: should be "you will receive an email" not "you will receive email"

Reply:
Changed